src/main/java/io/cdap/plugin/python/transform/JythonPythonExecutor.java

```diff
@@ -83,19 +81,10 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter,
       Py.runCode(compiledScript, interpreter.getLocals(), interpreter.getLocals());
 
     } catch (PyException e) {
-      // Put stack trace as the exception message, because otherwise the information from PyException is lost.
-      // PyException only exposes the actual cause (Python stack trace) if printStackTrace() is called on it.
-      throw new IllegalArgumentException("Could not transform input.\n" + getStackTrace(e));
+      throw new IllegalArgumentException("Encountered an error in the Python code.\n" + e.value);
```

Comment:
Isn't the `e` should be passed as the root cause?

Reply:
Should be just `throw new IllegalArgumentException("... some message... ", e);`

Reply:
PyException is weird. `e.getCause()` returns null.
![Screen Shot 2021-02-22 at 10 36 31 PM](https://user-images.githubusercontent.com/2873805/108809974-e38d5c80-755e-11eb-9d18-8d4cb9cca817.png)